source/examples/Migrations/PersonClassV1/PersonClassV1.kt

```diff
@@ -1,5 +1,4 @@
 class Person: RealmObject {
```

Comment:
```suggestion
// Realm schema version 1
class Person: RealmObject {
```

Thoughts on adding a comment like this to each relevant snippet?

i.e. identify the version at-a-glance

Reply:
This could definitely make things a little clearer, particularly for users who don't read the full page end-to-end. Added!